src/components/Input/Input.styled.ts

```diff
@@ -1,27 +1,31 @@
-import styled from 'styled-components';
-
+import styled, { DefaultTheme, ThemedStyledProps } from 'styled-components';
 import type { InputProps } from './types';
 
-const INPUT_WIDTH: Record<InputProps['size'], string> = {
+type MyThemedStyledProps = ThemedStyledProps<InputProps & DefaultTheme, DefaultTheme>;
+
+const INPUT_WIDTH: Record<InputProps['_size'], string> = {
   large: '100%',
   medium: '80%',
   small: '50%',
 };
 
-const Input = styled.input<InputProps>`
+const Input = styled.input<MyThemedStyledProps>`
```

Comment:
```suggestion
const Input = styled.inputt<Pick<InputProps, 'size'>>`
```
Pick 유틸리티타입으로 필요한것만 정의하는 것도 좋을 것 같아요.